### PR TITLE
Timing/Period functionality for game tracker

### PR DESCRIPTION
**Pull request by johnhof**
@DuaneGarber I want to get this in now. If I try to to it all at once it will be totally unreasonable to review.

The Gist:
- styling updates
- added
    -  scss mixins
    - `<stepper>` directive used to step integer values (may be useful later)
- implemented
    - timer and period transition functionality
    - dialog/modal styling (using ngDialog https://github.com/likeastore/ngDialog)
    - team select and display

to test
- go to `/#/tracker`
- select teams
- select teams or chose `default`
- check console that teams printed match teams selected
- test everything in the tracker header
- make sure teams are displaying
- test team variations and display

### CODE REVIEW

**Comment by DuaneGarber** [comment]
You committed a bunch of orig files that need to be removed

File: app/assets/directives/material/ripple/ripple.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    background: black;
+  }
+
+
+  animation: ripple-animation 1s;
+}
+
+
+@keyframes ripple-animation {
+  from {
```
Comment (DuaneGarber) [inline app/assets/directives/material/ripple/ripple.scss]: Should this be 0% and 100% instead of from and to?

**Comment by johnhof** [comment]
haha, sorry. This was prepped for review 3 days ago. I've been pushing code to this branch recently :p

**Comment by DuaneGarber** [comment]
Other wise looks good, how do you envision mobile device team selection to go?

I think it should be goal/penalty then team selection. ... not sure how that plays out for shots though, ultimately I dont want the user to have to press team, then goal, just seems not very intuitive

**Comment by johnhof** [comment]
@DuaneBrander  either flow works. I'll have same team select as the opening modal. it will default to the currently selected team

**Comment by johnhof** [comment]
@DuaneBrander bump for merge

**Pull request merged**